doc/man7/provider-signature.pod

```diff
@@ -470,8 +470,13 @@ The length of the "digest-size" parameter should not exceed that of a B<size_t>.
 
 =item "algorithm-id" (B<OSSL_SIGNATURE_PARAM_ALGORITHM_ID>) <octet string>
 
-Gets the DER encoded AlgorithmIdentifier that corresponds to the combination of
-signature algorithm and digest algorithm for the signature operation.
+Gets the DER-encoded AlgorithmIdentifier for the signature operation.
+This typically corresponds to the combination of a digest algorithm
+with a purely asymmetric signature algorithm, such as SHA256WithECDSA.
+
+The L<ASN1_item_sign_ctx(3)> relies on this operation and is used by
```

Comment:
This should probably be either

`The L<ASN1_item_sign_ctx(3)> function relies on this operation`

or

`L<ASN1_item_sign_ctx(3)> relies on this operation`

but it's not a major issue - happy whether you want to update or not

Reply:
Thanks @tom-cosgrove-arm for your review and this and the following improvement suggestion.
I've just added these nit fixes to #27367, as part of the commit improving `provider-signature.pod`.